setup/assets/js/sections/database.js

```diff
@@ -20,23 +20,23 @@ MODx.DB = function () {
           url: 'processors/connector.php',
           success: function (r) {
             r = Ext.decode(r.responseText);
-            var msg = Ext.select('#modx-db-step1-msg'),
-              dbInfo = Ext.select('#modx-db-info'),
-              step2 = Ext.select('#modx-db-step2');
+            const msg = Ext.select('#modx-db-step1-msg');
+            const dbInfo = Ext.select('#modx-db-info');
+            const step2 = Ext.select('#modx-db-step2');
```

Comment:
Not trying to nitpick, but for this block, wouldn't the preferred code style be this?

```
const msg = Ext.select('#modx-db-step1-msg'),
    dbInfo = Ext.select('#modx-db-info'),
    step2 = Ext.select('#modx-db-step2');
```

Reply:
I noticed that this approach to variable initialization is often used in the project.
This was OK when only `var` were used. But now the variables are initialized with `let` and `const`. Because of this, visually strange grouping can occur (fictional code example):

```javascript
const user = getUser(),
    ui = getUI()

let userId = getUserId(),
    output = ''
```

Therefore, it makes sense to group the initialization of variables by meaning, and not by type of variables:
```javascript
const user = getUser()
let userId = getUserId()

const ui = getUI()
let output = ''
```